src/main/g8/src/main/scala/$package$/UserRegistryActor.scala

```diff
@@ -1,7 +1,8 @@
-package com.lightbend.akka.http.sample
+package $package$
 
 import akka.actor.{ Actor, ActorLogging, Props }
 
+//#user-registry-actor
```

Comment:
Might be good to include the imports in the snippet?

Reply:
good point, addressed